README.md

```diff
@@ -44,30 +31,66 @@ The `.bake.yaml` configuration file can be stored in the following locations:
 
 Current configuration options and default values:
 
-- `projectRoot: "../"` : Directory of channel data files.    
+- `projectRoot: "../"` : Directory of channel data files.  
   E.g. `$GOPATH/src/github.com/breadtubetv/breadtubetv/data/channels`
 
-#### Importing a Channel
+### Commands
+
+Once you have `bake` installed:
+
+**Import a Channel:**
+
+```bash
+bake channel import creator_slug youtube channel_url
+```
 
-You can run bake directly from the source like so:
+**Import a Video:**
 
+```bash
+bake import video [--id | --url] video_id_or_URL --creators creator_slug --provider youtube
 ```
-go run main.go config youtube #follow prompts
-go run main.go channel import contrapoints youtube https://www.youtube.com/user/contrapoints
+
+## Contributing
+
+You'll need the Go programming language installed. We recommend version v1.12+. This is going to be dependent on your system, we recommend following <https://golang.org/doc/install>
+
+Clone the repo and pull dependencies:
+
+```bash
+# Enable Go 1.11 module support
+export GO111MODULE=on
+# Clone directory
+git clone https://github.com/breadtubetv/bake
+cd bake
+# Download dependencies
+go mod download
 ```
 
-Or if you've installed it:
+> **NOTE:** The `spf13/cobra` generator CLI won't work if you don't clone the project into `$GOPATH`. This is not a requirement to develop for the project.
+
+### Linting & Pre-Commit
+
+We recommend using [`golangci/golangci-lint`](https://github.com/golangci/golangci-lint) for linting. A development config (`.golangci.dev.yaml`) is provided. To use this config:
 
+```shell
+golangci-lint run --config=<path/to/.golangci.dev.yaml>
 ```
-bake config youtube #follow prompts
-bake channel import contrapoints youtube https://www.youtube.com/user/contrapoints
+
+A `.pre-commit-config.yaml` is provided for pre-commit hooks using [`pre-commit/pre-commit`](https://github.com/pre-commit/pre-commit).
+
+### Submitting a PR
+
+We welcome PRs! The only thing we ask is that you ensure you keep the `go.mod` and `go.sum` files clean by running the following:
+
+```shell
+GO111MODULE=on go mod tidy
 ```
 
-#### Testing
+### Testing
```

Comment:
Would good to have some information for developers about how to use https://goreleaser.com/, eg

```suggestion
### Releasing

- `brew install goreleaser`
- Create a tag
- Run `goreleaser`
```